Dockerfile

```diff
@@ -7,14 +7,11 @@ FROM golang:1.12.7-alpine3.10 AS builder
 
 # We need so that dep can fetch it's dependencies
 RUN apk --no-cache add git
-RUN go get github.com/golang/dep/cmd/dep
 
 # Directory in workspace
 WORKDIR "/go/src/github.com/Peripli/service-manager"
 
-# Copy dep files only and ensure dependencies are satisfied
-COPY Gopkg.lock Gopkg.toml ./
-RUN dep ensure --vendor-only -v
+RUN go mod vendor
```

Comment:
Why do you need this?

go build will work just fine even if you don't create the vendor directory.